stories/src/multiple-vertical/quote-app.jsx

```diff
@@ -121,8 +128,19 @@ export default class QuoteApp extends Component {
     `;
   }
 
+  getDisabledDroppable = (sourceDroppable: ?string) => {
```

Comment:
I updated the stress test story to demo this - when you lift from a droppable the next droppable will dynamically become disabled.

Reply:
I was thinking of creating a little button where we can set the disabled / enabled status of a droppable in the story - don't need to do it here though. Thoughts?

Reply:
we can leave this for now though - we can build on it

Reply:
yeah that's something we can add later when we polish the stories IMO - this is just a quick and dirty way to test it now :)